draft-thomson-httpbis-alt-svcb.md

```diff
@@ -343,20 +402,26 @@ than "example.com", clients will not use this service unless an alternative was
 provided by the server:
 
 ~~~ dns
-alt1.example. 7200 IN HTTPS 1 . port=443,alt-only,mandatory=alt-only
+alt1.example. 7200 IN HTTPS 1 . port=443 alt-only mandatory=alt-only
 example.com.  7200 IN HTTPS 2 . port=443
 ~~~
 
-\[\[ISSUE: Do we need this flag in addition to the priority override?  Could one
-    of the two be sufficient?]]
+
+## Servers Identified by IP {#ip}
+
+An alternative name can be provided by a server that is identified by an IP
+address or host names that are not domain names.  However, HTTPS queries cannot
+be made for servers that are not identified by a domain name.  This makes reuse
+({{reuse}}) impossible for such names.  A client MAY disable alternative
```

Comment:
Not just reuse -- you can't point to them for the current session. It makes un-named servers impossible to reference as alternatives.